proto/build.rs

```diff
@@ -21,6 +21,7 @@ fn main() -> miette::Result<()> {
     // Generate the stub of the user facing server from its proto file
     tonic_build::configure()
         .file_descriptor_set_path(&file_descriptor_path)
+        .type_attribute(".", "#[derive(Eq, PartialOrd, Ord, Hash)]")
```

Comment:
`.` means `all generated classes`. So this is adding the derive to all the message types.